parm/config/config.anal

```diff
@@ -53,25 +53,6 @@ export OBERROR=${FIXgsi}/prepobs_errtable.global
 
 # Use experimental dumps in EMC GFS v16 parallels
 if [[ ${RUN_ENVIR} == "emc" ]]; then
-  export ABIBF="/dev/null"
-  if [[ "${CDATE}" -ge "2019022800" ]] ; then
-    export ABIBF="${COMIN_OBS}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
-  fi
-
-  export AHIBF="/dev/null"
-  if [[ "${CDATE}" -ge "2019042300" ]]; then
-    export AHIBF="${COMIN_OBS}/${CDUMP}.t${cyc}z.ahicsr.tm00.bufr_d"
-  fi
-
-  export HDOB="${COMIN_OBS}/${CDUMP}.t${cyc}z.hdob.tm00.bufr_d"
-
-  #   Use dumps from NCO GFS v16 parallel
-  if [[ "${CDATE}" -ge "2020103012" ]]; then
-    export ABIBF=""
-    export AHIBF=""
-    export HDOB=""
-  fi
-
```

Comment:
Did this end up somewhere else or just gone?

Reply:
Just gone. Rahul said ditch it.